.github/workflows/notify.yml

```diff
@@ -0,0 +1,55 @@
+name: Notify on Workflow Complete
+
+on:
+  workflow_run:
+    workflows: [Build and Release]
```

Comment:
Will this need to be updated once #84 is merged?

Reply:
Yes. We need to add `verify.yml `in the flows.